sdk/storage/azure-storage-file/azure/storage/file/file_client.py

```diff
@@ -1057,11 +1046,12 @@ def clear_range( # type: ignore
         if self.require_encryption or (self.key_encryption_key is not None):
             raise ValueError("Unsupported method for encryption.")
 
-        if start_range is None or start_range % 512 != 0:
-            raise ValueError("start_range must be an integer that aligns with 512 file size")
-        if end_range is None or end_range % 512 != 511:
-            raise ValueError("end_range must be an integer that aligns with 512 file size")
-        content_range = 'bytes={0}-{1}'.format(start_range, end_range)
+        if offset is None or offset % 512 != 0:
+            raise ValueError("offset must be an integer that aligns with 512 file size")
+        if length is None or length % 512 != 511:
```

Comment:
`if length is None or length % 512 != 0:`
or 
`end_range = length + offset - 1`
`if end_range is None or end_range % 512 != 511:`